Documentation~/TableOfContents.md

```diff
@@ -100,9 +100,8 @@
 	* [Triangulate - Objects](Object_Triangulate)
 	* [Weld Vertices](Vert_Weld)
 * [Tools reference](ref_tools)
-	* [Cut tool](cut-tool)
-	* [Poly Shape tool](polyshape)
-	* [Shape tool](shape-tool)
+	* [New PolyShape and Edit PolyShape tools](polyshape)
+	* [New Shape and Edit Shape tools](shape-tool)
```

Comment:
I think "New" at the beginning of these entries is just setting ourselves up for more maintenance work in the future and, potentially, user confusion. We could always just link to them in the What's New page to indicate that they're newer.

Reply:
It's new v edit, not new v old